FirebasePerformance/Tests/TestApp/README.md

```diff
@@ -3,9 +3,13 @@
 ## Setup
 
 Download the `GoogleService-Info.plist` file from [Firebase Console](https://console.firebase.google.com/)
-(for whatever Firebase project you have or want to integrate the `dev-app`). For Autopush environment, store the
+(for whatever Firebase project you have or want to integrate the `dev-app`).
+
+For Prod environment, store the `GoogleService-Info.plist` under [Plists/Prod/FIRPerfTestApp/](./Plists/Prod/FIRPerfTestApp/). This should be sufficient for most scenarios.
+
+For Autopush environment, store the
```

Comment:
Lets qualify that the events generated for the Autopush environment will not be available on the console outside of Google as these are processed on our staging servers.

Reply:
Added.